lib/integration/queryManager.ts

```diff
@@ -0,0 +1,30 @@
+import { logMessage } from "../logger";
+import dbConnector from "./dbConnector";
+import { QueryArrayResult } from "pg";
+
+const queryManager = {
+  async executeQuery(sql: string, params: [string]): Promise<QueryArrayResult<unknown[]>> {
+    try {
+      // Get a client
+      const client = dbConnector();
+      //Stablish a connexion
+      await client.connect();
+      if (!sql) throw new Error("Invalid query param");
```

Comment:
This should be moved up before everything in the function. Please work on writing clearer error messages. If I'm seeing this in the logs it would be hard to tell where this came from or would take a lot more effort. Something like 

`Empty string passed in to queryManager` would be a lot more clearer